src/main/webapp/resources/js/Controller/applicantController.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+angular.module('applicantApp')
+.controller('mainController',['$http','applicantUtilService','$scope',function($http,$scope,applicantUtilService,$location,$sce){
+	
```

Comment:
I think parametyer `applicantUtilService` ==> `$applicantUtilService` in function parameter